pypeit/wavecalib.py

```diff
@@ -56,8 +56,10 @@ class WaveCalib(masterframe.MasterFrame):
         steps
     arccen : ndarray (nwave, nslit)
       Extracted arc(s) down the center of the slit(s)
-    maskslit : ndarray (nslit)
-      Slits to ignore because they were not extacted
+    maskslits : ndarray (nslit); bool
+      Slits to ignore because they were not extracted
+      WARNING: Outside of this Class, it is best to regenerate
+      the mask using  _make_maskslits()
```

Comment:
If we use (or recommend usage of) `_make_maskslits` outside of the class, we should probably not make it a "private" method.  I.e., we should change the name to `make_maskslits`.

Reply:
yeah, good call.  Am modifying that now.